internal/cssc/cssc.go

```diff
@@ -24,19 +27,42 @@ import (
 	"oras.land/oras-go/v2/registry/remote/retry"
 )
 
+type TagConvention string
+
+const (
+	Incremental TagConvention = "incremental"
+	Floating    TagConvention = "floating"
+)
+
+func (tc TagConvention) IsValid() error {
+	switch tc {
+	case Incremental, Floating:
+		return nil
+	}
+	return errors.New("tag-convention should be either incremental or floating")
+}
+
+var (
+	floatingTagRegex    = regexp.MustCompile(`^(.+)-patched`)             // Matches strings ending with -patched
```

Comment:
As a note this doesn't match "-patched" itself. I assume this is expected but just noting it

Reply:
yes correct